pkg/certificate/providers/config_test.go

```diff
@@ -30,8 +29,8 @@ import (
 
 func TestGetCertificateManager(t *testing.T) {
 	mockCtrl := gomock.NewController(t)
-	computeMock := compute.NewMockInterface(mockCtrl)
-	computeMock.EXPECT().GetMeshConfig().AnyTimes()
+	MockInfraClient := NewMockProvidersInfraClient(mockCtrl)
```

Comment:
this seems simple enough to avoid mock generation as well. Can we create a simple struct here in its place?